src/main/java/org/folio/circulation/resources/CheckOutByBarcodeResource.java

```diff
@@ -178,6 +197,42 @@ private CompletableFuture<Result<LoanAndRelatedRecords>> saveCheckOutSessionReco
         succeeded(records)));
   }
 
+  private CompletableFuture<Result<LoanAndRelatedRecords>> acquireLock(LoanAndRelatedRecords records, CheckOutLockRepository checkOutLockRepository, StringBuilder checkOutLockId) {
+    log.debug("acquireLock:: Creating checkout lock {} ",records.getCheckoutLockConfiguration());
+    if(records.getCheckoutLockConfiguration() == null || !records.getCheckoutLockConfiguration().isCheckOutLockFeatureEnabled()) {
```

Comment:
Please follow widely accepted coding conventions for whitespace. Here you need a space after `if`. But on the line above you need a space after the comma on your params. Below in many places you need a space before the open brace after a keyword.

For a guide to whitespace I think the google standard is easiest to follow: https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Reply:
Added spaces wherever necessary. Hence resolving all spacing related comments.